### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by odaialazzeh**
- [ ] Update the HTML page and add the main container.
- [ ] Create a CSS file and add styles for the components.
- [ ] Create calculater.js which contains the UI design.

### CODE REVIEW

File: src/calculator.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+const Screen = () => (
+  <div className="screen-row">
+    <input type="text" value={0} readOnly />
+  </div>
+);
+
+Button.propTypes = {
+  label: PropTypes.string,
+};
```
Comment (thecodechaser) [inline src/calculator.js]: - [ ] Please create a directory called `components` and move your `calculator` component to the directory as requested in the project requirements.

image

File: src/calculator.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  return (
+    <button type="button">{label}</button>
+  );
+};
+
+const Screen = () => (
+  <div className="screen-row">
+    <input type="text" value={0} readOnly />
+  </div>
+);
```
Comment (thecodechaser) [inline src/calculator.js]: - [ ] Please make sure that all components are class-based as requested in the project requirements.

image

Comment (thecodechaser) [inline src/calculator.js]: - [ ] [optional] I would suggest moving these components to a separate files, It is a good practice to keep components in separate files.

File: src/calculator.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+const Button = (props) => {
+  const { label } = props;
+  return (
+    <button type="button">{label}</button>
+  );
+};
+
+const Screen = () => (
+  <div className="screen-row">
+    <input type="text" value={0} readOnly />
```
Comment (thecodechaser) [inline src/calculator.js]: - [ ] Please make sure that your app has a component `App` that will be the starting point(main component) of the application and call the `calculator` component in the `App` component.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @odaialazzeh👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @odaialazzeh,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**